src/components/Notes/NewNoteModal/new-note-modal.css

```diff
@@ -10,10 +10,19 @@
 
 .new-note-form {
     max-width: 500px;
```

Comment:
you can use rem here

Reply:
I actually have reduced the font size of the root as the window screen size decreases. Because of this, I did not use rem for length as I did not want the length to reduce.